src/renderer/state/upload/test/selectors.test.ts

```diff
@@ -149,6 +150,57 @@ describe("Upload selectors", () => {
       const actual = getUploadRequests(state);
       expect(actual).to.deep.equal(expectedPayload);
     });
+    it("Interprets combined 'File Name (File ID)' FMS.File Lookup values into just File IDs", () => {
+      const state: State = {
+        ...nonEmptyStateForInitiatingUpload,
+        template: {
+          ...nonEmptyStateForInitiatingUpload.template,
+          appliedTemplate: {
+            ...mockMMSTemplate,
+            annotations: [mockFmsFileLookupAnnotation],
+          },
+        },
+        upload: getMockStateWithHistory({
+          "/path/to.dot/image.tiff": {
+            FileLookupAnnotation: [
+              "File 1 (AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA)",
```

Comment:
in response to Phillips point, maybe a test where the string is
"XAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAX (AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA)"
and make sure you get the Second value back?
For that matter, you might want to test 
"File 1 (AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA)"  - short a character
"File 1 AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA" missing ()
"File 1 (AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA)" extra character

Reply:
Mm yeah, those are good ideas. I'll add some with varying file name lengths too.